ee/ucp/kubernetes/layer-7-routing.md

```diff
@@ -19,48 +19,8 @@ Use an ingress controller when you want to:
 * Give your Kubernetes app an externally-reachable URL.
 * Load-balance traffic to your app.
 
-Kubernetes provides an NGINX ingress controller that you can use in Docker EE
-without modifications.
-Learn about [ingress in Kubernetes](https://v1-8.docs.kubernetes.io/docs/concepts/services-networking/ingress/).
-
-## Create a dedicated namespace
-
-1.  Navigate to the **Namespaces** page, and click **Create**.
-2.  In the **Object YAML** editor, append the following text.
-    ```yaml
-    metadata:
-      name: ingress-nginx
-    ```
-
-    The finished YAML should look like this.
-
-    ```yaml
-    apiVersion: v1
-    kind: Namespace
-    metadata:
-      name: ingress-nginx
-    ```
-3.  Click **Create**.
-4.  In the **ingress-nginx** namespace, click the **More options** icon,
-    and in the context menu, select **Set Context**.
-
-    ![](../images/deploy-ingress-controller-1.png){: .with-border}
-
-## Create a grant
-
-The default service account that's associated with the `ingress-nginx`
-namespace needs access to Kubernetes resources, so create a grant with
-`Restricted Control` permissions.
-
-1.  From UCP, navigate to the **Grants** page, and click **Create Grant**.
-2.  Within the **Subject** pane, select **Service Account**. For the
-    **Namespace** select **ingress-nginx**, and select **default** for
-    the **Service Account**. Click **Next**.
-3.  Within the **Role** pane, select **Restricted Control**, and then click
-    **Next**.
-4.  Within the **Resource Set** pane, select the **Type** **Namespace**, and
-    select the **Apply grant to all existing and new namespaces** toggle.
-5.  Click **Create**.
+Kubernetes provides an NGINX ingress controller that can be used in Docker EE, but is not officially supported. Modifications are typically required based on your environment.
```

Comment:
Can we change this line to:

>This example serves as a recommendation for how to deploy an ingress controller on Docker Enterprise. It is based on the Kubernetes community [NGINX controller](https://github.com/kubernetes/ingress-nginx) and runs on Docker Enterprise. It is, however, not officially a component of Docker Enterprise and is not directly supported by Docker Inc.

Reply:
@mark-church - I shortened your statement a bit - please make sure it captures what should be conveyed - thank you!